src/java/org/apache/cassandra/journal/ActiveSegment.java

```diff
@@ -22,13 +22,16 @@
 import java.nio.MappedByteBuffer;
 import java.nio.channels.FileChannel;
 import java.nio.file.StandardOpenOption;
-import java.util.*;
+import java.util.Set;
```

Comment:
Leave all this imports changes out of the PR for hygiene? No code changes in this file.